app/assets/javascripts/file_manager.js

```diff
@@ -0,0 +1,137 @@
+function rename(path) {
+    let new_name = prompt("Enter the new name:");
```

Comment:
Should check if this returns `null` (i.e. the user clicked the "cancel" button)

Reply:
fixed.